index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Architectural Summit | Home</title>
+  <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+  <header>
+    <ul class="social">
+      <li>
+        <a href="http://web.facebook.com/lawrence.amoafo"><ion-icon name="logo-facebook" id="icon"></ion-icon></a>
+      </li>
+      <li>
+        <a href="http://twitter.com/lawrence_amoafo"><ion-icon name="logo-twitter" id="icon"></ion-icon></a>
+      </li>
+      <li>English</li>
+      <li>My Page</li>
+      <li>Logout</li>
+    </ul>
+
+    <nav class="head">
+      <div class="vector">
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
+
+      <div class="logo">
+        <a href="./index.html"><img src="./Images/logo.png" alt="logo" /></a>
+        <a href="./index.html"><b>World Architectural Summit 2023</b></a>
+      </div>
+
+      <ul class="nav-links">
+        <li><a href="#">Home</a></li>
+        <li><a href="pages/about.html">About</a></li>
+        <li><a href="#program">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#speakers">Speakers</a></li>
+      </ul>
+
+      <ul class="nav-links2">
+        <li><a href="#">Home</a></li>
+        <li><a href="pages/about.html">About</a></li>
+        <li><a href="#program">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#speakers">Speakers</a></li>
+        <li class="style">WA Summit</li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class="home" id="home">
+      <div class="place">THE OBEROI / DUBAI / UAE</div>
+      <h1>
+        Sustainable design, resilient transportation and empowered youth - the
+        keys to a brighter future
+      </h1>
+      <div class="date"><b>1 June 2023</b></div>
+    </section>
+
+    <section class="program" id="program">
+      <h2>Main Program</h2>
+
+      <span class="line"></span>
+
+      <div class="container1">
+        <div class="card">
```

Comment:
- [ ] Kindly consider taking the `article` tag for cards instead of `div` for better semantics and best practices here and in all similar situations.